src/views/PuzzleView.vue

```diff
@@ -128,6 +129,12 @@ export default Vue.extend({
         },
         roadmap(): Achievement[] {
             return this.$store.state.roadmap;
+        },
+        title(): string{
+            return DOMPurify.sanitize(this.puzzle.title);
+        },
```

Comment:
You shouldn't need to run the title through DOMPurify, as Vue will sanitize it for us (we're just interpolating it, not using v-html, as we should - markup makes sense in the description, but not the title)

Reply:
Good point. Fixing.